ethsigner/src/main/java/tech/pegasys/ethsigner/CommandLineConfig.java

```diff
@@ -196,4 +196,32 @@ public ChainIdProvider getChainId() {
   public Duration getDownstreamHttpRequestTimeout() {
     return Duration.ofMillis(downstreamHttpRequestTimeout);
   }
+
+  @Override
+  public String toString() {
+    return "CommandLineConfig{"
```

Comment:
Not a fan of the `StringBuilder` then?

Reply:
Our typical approach is to use guava's `MoreObjects.toStringHelper` for `toString` methods.  `StringBuilder` should only be used when there's a loop involved - if the list of strings to concatenate is static you may as well let the compiler generate all the `.append` calls for you and stick with the easier to read plain string concatenation.

Reply:
:( This is the pre-built intellij code - might need a new template

Reply:
Fortunately Intellij Provides the "MoreObjects.toStringHelper (Guava 18+)" template which comes highly recommended by the Pantheon coding conventions (https://github.com/PegaSysEng/pantheon/blob/master/CODING-CONVENTIONS.md#423-common-methods) :)

Reply:
yeah, did find that :)